wasm-corpus/pom.xml

```diff
@@ -18,4 +18,34 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>3.5.0</version>
+        <executions>
+          <execution>
+            <id>generate-wat</id>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+            <phase>install</phase>
+            <configuration>
+              <outputFile>${project.basedir}/src/main/resources/wat/functions_10.wat</outputFile>
+              <executable>java</executable>
+              <arguments>
+                <argument>-cp</argument>
+                <classpath/>
+                <argument>com.dylibso.chicory.corpus.WatGenerator</argument>
```

Comment:
Since we are committing the result I believe that this build step is redundant.

Reply:
What if we change the wat generator?   We would want to it regenerate right?

Reply:
Leaving a note on this `main` would be enough for me:

https://github.com/dylibso/chicory/pull/833/files#diff-9810e9da2860cd11cffd4037c2c2be942518f0f09a3a57634076676db41e75e2R51

Alternatively, `exec-maven-plugin.version` goes in the root `pom.xml`, the dependency listed etc. etc.